db/migrate/20180206102132_change_column_default.rb

```diff
@@ -0,0 +1,12 @@
+class ChangeColumnDefault < ActiveRecord::Migration[5.1]
```

Comment:
The code is not indented properly. Please indent it.

Reply:
indentation issue fixed